components/worktemplates/utils.ts

```diff
@@ -3,31 +3,31 @@
 
 import {WorkTemplate} from '@mattermost/types/worktemplates';
 
-export function getTemplateDefaultIllustration(template: WorkTemplate): string {
+export function getTemplateDefaultIllustration(template: WorkTemplate): [string, string] {
     const channels = template.content.filter((c) => c.channel).map((c) => c.channel!);
     if (channels.length) {
-        return channels[0].illustration;
+        return ['channels', channels[0].illustration];
```

Comment:
nit: good practice to avoid magic strings, use an enum to hold these values is a better practice.